### PR TITLE
Made all the tests pass in the quiz

### PR DESCRIPTION
**Pull request by MarkZaWill**

### CODE REVIEW

**Comment by chortlehoort** [comment]
@MarkZaWill 

1. Your logic all looks good, but putting all your classes in one file is a big no-no. Each class gets its own file.
1. It's also best practice to name the project appropriately. It should not be ConsoleApplication1 and there should never be a file named `Class1.cs`.

Mark, these may seem nit-picky, but I'm making sure you do this now, and get in the practice of it because if you submitted this on the job, you'd get torn apart by your teammates and/or your manager in a code review.

**Comment by MarkZaWill** [comment]
I understand what you mean.  Having changed this around, it is Waaaaaaaaay more organized.  Thanks!

File: ConsoleApplication1/QuizObjects/FineDining.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LunchQuizTests
```
Comment (chortlehoort) [inline ConsoleApplication1/QuizObjects/FineDining.cs]: @MarkZaWill One last thing.  The namespace for your implementation project should be different than the namespace for your tests. Change this namespace in the implementation project and you're done.

**Comment by MarkZaWill** [comment]
Done!

**Comment by chortlehoort** [comment]
👍 

Merge it!

**Comment by MarkZaWill** [comment]
Hooray!!!

**Pull request merged**